OpenRA.Mods.Cnc/Traits/Minelayer.cs

```diff
@@ -24,7 +24,7 @@ namespace OpenRA.Mods.Cnc.Traits
 {
 	public class MinelayerInfo : TraitInfo, Requires<RearmableInfo>
 	{
-		[ActorReference]
+		[ActorReference(typeof(MineInfo))]
```

Comment:
As far as I can tell, this trait / order generators don't require this actor to have the `Mine` trait?

Reply:
You are right. This doesn't fix a crash or anything.